src/shaders/brdf.glsl

```diff
@@ -1,4 +1,13 @@
 
+// Compute the achromatic normal incidence reflectance based on a given index of refraction of a dielectric material.
+// The index of refraction of the surrounding is assumed to be 1.0, which is approximatly the case for air.
+// For an index of refraction of 1.5, this results the known default value of 0.04 reflectivity for dielectrics.
+float f0_ior(float ior)
+{
+    float f0 = sq((ior - 1.0) / (ior + 1.0));
```

Comment:
Since IOR is `uniform`, this per-fragment computation seems expensive.  Could the app compute this value and pass it in as uniform?

Reply:
Yes ... and this is the reason why I do not want this extension for reflection, as this information is already available in the Metallic-Roughness material - or for more control - in the specular one. This would be the third(!) possibility to define F0.

Reply:
But please compare to https://github.com/KhronosGroup/glTF/pull/1718 and https://github.com/KhronosGroup/glTF/pull/1719 as well.